crates/autopilot/src/arguments.rs

```diff
@@ -479,6 +493,24 @@ impl FromStr for FeePolicy {
     }
 }
 
+#[derive(clap::Parser, Clone)]
+pub struct CircuitBreaker {
+    pub circuit_breaker_authenticator_pk: Option<H256>,
+    pub circuit_breaker_solvers: Option<Vec<H160>>,
+}
+
+impl Debug for CircuitBreaker {
+    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        f.debug_struct("CircuitBreaker")
+            .field(
+                "circuit_breaker_authenticator_pk",
+                &self.circuit_breaker_authenticator_pk.map(|_| "***"),
```

Comment:
This is a secret, so it shouldn't be printed. Is it ok to read this from a file? should we fetch this directly from KMS as @fleupold suggested here https://github.com/cowprotocol/services/pull/2705#discussion_r1603634984?

Reply:
@MartinquaXD what is the practise you guys use here, should we specify an ARN in the config instead of the PK and the ARN corresponds with the secret in KMS?

Reply:
Very likely I could address this in another PR :thinking:

Reply:
Yeah, we would configure the `ARN` instead. This logic already exists in the driver so you could check there how it has to be done.